### PR TITLE
fix #77372 hrdergi.com

### LINKED ISSUE #77372: hrdergi.com
### Issue URL (Ads)
 [https://hrdergi.com/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fhrdergi.com%2F)
### Comment
Username: @RadicalNoise
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.31
Browser: | Chrome
Stealth mode options: | Hide your search queries, <br>Remove X-Client-Data header from HTTP requests, <br>Strip URLs from tracking parameters, <br>Self-destructing third-party cookies (2880)
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Security:</b><br/> NoCoin Filter List<br/><br/><b>Language-specific:</b><br/> AdGuard Turkish

### PR DESCRIPTION
**Pull request by wout-codes**
#77372

### CODE REVIEW

File: TurkishFilter/sections/specific.txt [hunk 1]
```diff
@@ -42,6 +42,7 @@ yemektarifleri.org,lifebursa.com,mshowto.org,macizlee1.com,macizlee.ne,dunyaatla
 fullhdfilmcibaba.com,720pizlemek.com,1080pfilmizle.net,fullhdfilmcidayi.com,hdfilmizledayi.com,720-pizleme.com,fullhdfilmizlebaba.com,evrenselfilmlerim.net###adStarter
 !
 forum.donanimhaber.com##div[id^="dha_"]
+hrdergi.com##a[href^="https://globalhrsummit.org"]
```
Comment (Alex-302) [inline TurkishFilter/sections/specific.txt:45]: it seems `a[data-ad-id]` is better

Comment (wout-codes) [reply TurkishFilter/sections/specific.txt:45]: It doesn't work.

Comment (Alex-302) [reply TurkishFilter/sections/specific.txt:45]: weird. I hope you checked it with `##`?:) `hrdergi.com##a[data-ad-id]`
image

Comment (wout-codes) [reply TurkishFilter/sections/specific.txt:45]: Yeah, it works somehow now.

**Review by zloyden — APPROVED** [review]

**Pull request merged**